src/Humanize/Languages/EnglishHumanizer.php

```diff
@@ -76,8 +77,20 @@ private function humanizeSeason( Season $season ): string {
 		return self::SEASON_MAP[$season->getSeason()] . ' ' . $season->getYear();
 	}
 
+    /**
+     * @psalm-suppress MixedArgument
+     */
 	private function humanizeDate( ExtDate $date ): string {
-		$humanizedDate = $this->humanizeYearMonthDay( $date->getYear(), $date->getMonth(), $date->getDay() );
+	    if ($date->unspecified()) {
+	        $unspecified = $date->getUnspecifiedDigit();
+        }
+
+		$humanizedDate = $this->humanizeYearMonthDay(
+		    $date->getYear(),
+            $date->getMonth(),
```

Comment:
PSR-2 is one thing... but using spaces instead of tabs?

![image](https://user-images.githubusercontent.com/146040/107509542-0012b780-6ba3-11eb-825d-33eed7f14095.png)